.github/workflows/cont_int.yml

```diff
@@ -117,6 +117,7 @@ jobs:
         - name: Cythonize RMG-Py
           run: |
             cd RMG-Py
+            conda activate rmg_env
```

Comment:
this line was removed in another commit and added here. Can you squash the two commits?